[2023-01-25 23:15:17,056] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-25 23:15:17,073] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-25 23:15:17,073] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:15:17,074] {taskinstance.py:1377} INFO - Starting attempt 3 of 5
[2023-01-25 23:15:17,074] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:15:17,096] {taskinstance.py:1397} INFO - Executing <Task(GeneratesatelliteDataOperator): generate_platzi_data> on 2023-01-03 00:00:00+00:00
[2023-01-25 23:15:17,103] {standard_task_runner.py:52} INFO - Started process 3519 to run task
[2023-01-25 23:15:17,110] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'generate_platzi_data', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '473', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpcpe30dz0', '--error-file', '/tmp/tmp67xa2td7']
[2023-01-25 23:15:17,113] {standard_task_runner.py:80} INFO - Job 473: Subtask generate_platzi_data
[2023-01-25 23:15:17,225] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [running]> on host fd9891b969b0
[2023-01-25 23:15:17,350] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=generate_platzi_data
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-03T00:00:00+00:00
[2023-01-25 23:15:17,350] {logging_mixin.py:115} INFO - La fecha l√≥gica es: 20230103
[2023-01-25 23:15:17,356] {logging_mixin.py:115} INFO - archivo creado en: /tmp/platzi_data_20230103.csv
[2023-01-25 23:15:17,374] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=generate_platzi_data, execution_date=20230103T000000, start_date=20230125T231517, end_date=20230125T231517
[2023-01-25 23:15:17,407] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-25 23:15:17,442] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
